tools/SetupFlow/DevHome.SetupFlow/ViewModels/MainPageViewModel.cs

```diff
@@ -29,6 +29,8 @@ namespace DevHome.SetupFlow.ViewModels;
 /// </summary>
 public partial class MainPageViewModel : SetupPageViewModelBase
 {
+    private const string _hideSetupFlowBannerKey = "HideSetupFLowBanner";

```

Comment:
Tiny thing -- typo in capitalization
```suggestion
    private const string _hideSetupFlowBannerKey = "HideSetupFlowBanner";

```